scripts/linux/4_boot_leds.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/bash
 
-rm -rf /home/xilinx/pynq/bitstream/.log
```

Comment:
@PeterOgden  This is fine. We don't have to delete it again since `3_pl_server.sh` already does it.